packages/extension/src/newtab/CardSelection.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
+import classNames from 'classnames';
+import VIcon from '@dailydotdev/shared/icons/v.svg';
+import React, { ReactElement, ReactNode } from 'react';
+
+interface CardSelectionProps {
+  title: string;
+  description: string;
+  icon: ReactNode;
+  isActive?: boolean;
+  onClick?: () => unknown;
+}
+
+export function CardSelection({
+  title,
+  description,
+  icon,
+  isActive,
+  onClick,
+}: CardSelectionProps): ReactElement {
+  return (
+    <button
+      className={classNames(
+        'relative flex flex-col gap-1 py-3 px-4 items-center border border-theme-divider-tertiary rounded-16 hover:cursor-pointer',
+        isActive
+          ? 'border-theme-divider-primary'
+          : 'border-theme-divider-tertiary',
+      )}
+      onClick={onClick}
+      type="button"
+    >
+      {isActive && (
+        <Button
```

Comment:
I think the `iconOnly` shouldn't be necessary anymore since the deciding factor is the computed value inside (also I think the prop is not being used anywhere).

https://github.com/dailydotdev/apps/blob/8479fcbc05149818870eca7ef5b4f6be32b87d7f/packages/shared/src/components/buttons/Button.tsx#L58